docs/tutorials/prowler-app-mute-findings.md

```diff
@@ -0,0 +1,53 @@
+# Mute Findings (Mutelist)
+
+Prowler App allows users to mute specific findings to focus on the most critical security issues. This comprehensive guide demonstrates how to effectively use the mutelist feature to manage and prioritize security findings.
+
+## What Is the Mutelist Feature?
+
+The mutelist feature enables users to:
+* **Suppress specific findings** from appearing in future scans
+* **Focus on critical issues** by hiding resolved or accepted risks
+* **Maintain audit trails** of muted findings for compliance purposes
+* **Streamline security workflows** by reducing noise from non-critical findings
```

Comment:
Review this line because indentation is not working.

<img width="846" height="203" alt="Image" src="https://github.com/user-attachments/assets/59853cad-5d11-421a-824d-79b502599954" />

See https://prowler-prowler-docs--8201.com.readthedocs.build/projects/prowler-open-source/en/8201/tutorials/prowler-app-mute-findings/#what-is-the-mutelist-feature